.github/workflows/update_backend.yml

```diff
@@ -28,39 +31,57 @@ jobs:
           git clone https://github.com/MilesCranmer/SymbolicRegression.jl
           cd SymbolicRegression.jl
           echo "version=$(git describe --tags --abbrev=0 | sed 's/^v//')" >> $GITHUB_OUTPUT
+          echo "version=$(git describe --tags --abbrev=0 | sed 's/^v//')"
```

Comment:
Why was this one changed?

Reply:
Debugging this workflow with no output pretty hard, so I added echos throughout so the different versions are in the logs. You're welcome to remove them of course.

Reply:
Oh wait, sorry, I see that the version is now stored in `pysr/version.py`, so this part actually isn't needed?